docs/client_guide.md

```diff
@@ -25,7 +25,7 @@ After determining the Bundle CID, client should supply a local path for writing
 
 ### CLI example
 ```
-➜  file-exchange git:(main) ✗ cargo run -p file-exchange downloader \
+$ fhs downloader \
```

Comment:
I'm not sure about using `fhs` as the CLI command :thinking: `file-hosting`, `graph-file`...?